src/sky.cpp

```diff
@@ -228,13 +227,36 @@ void Sky::render()
 			driver->drawIndexedTriangleFan(&vertices[0], 4, indices, 2);
 		}
 
-		// Draw far cloudy fog thing
+		// Draw middle band of our smooth skybox
```

Comment:
What's the purpose of this since it's identical in colour to `// Draw bottom sides of the sky`, the 2 operations can be combined into 1.

Reply:
Irrlicht's rendering is needed on all four sides when the sun, moon, stars aren't visible, IE, when you're in a cave or indoors.

Reply:
I don't see the relevance of that.
These 4 quads seem identical in code to the next 4 quads, so surely can be combined?

Reply:
Yes, it just makes sure that grey skybox looks the same as it would be outside (if the sky was a similar colour with the sun and moon visible, so to speak).

Reply:
Ok i probably misunderstand, is it because it needs to match the alpha colours added later to the East and West?